src/adapters/postgres/fields-adapter.ts

```
@@ -1217,31 +1220,31 @@ export class PostgresFieldsService implements IServicelocatorfields {
           offset,
           limit,
           order,
-          optionName
+          optionName,
         );
-      } else if (fetchFieldParams?.sourceDetails?.source === "jsonFile") {
+      } else if (fetchFieldParams?.sourceDetails?.source === 'jsonFile') {
```

ESLint

[error] Replace `'jsonFile'` with `"jsonFile"`

(prettier/prettier)